buf/registry/module/v1beta1/commit_service.proto

```diff
@@ -250,10 +250,12 @@ message GetCommitNodesRequest {
     //
     // The path must be relative, and connect contain any "." or ".." components
     // The separator "/" must be used.
-    //
-    // TODO: Can we encode the above requirements via validation?
     repeated string paths = 2 [(buf.validate.field).repeated.items = {
-      string: {max_len: 4096}
+      string: {
+        max_len: 1000
```

Comment:
Why 1000? I think this is a breaking change to the Buf CLI, and we chose 4096 as it had to do with Windows (I believe 4096 was the max file length).

Reply:
4096 SGTM. Not the max on Win32 (complicated) but the max in Git and both are far larger than any path we've seen yet.